manifests/w/WiseCleaner/WiseImageX/1.5.1/WiseCleaner.WiseImageX.installer.yaml

```
@@ -0,0 +1,35 @@
+# Created with komac v2.12.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.10.0.schema.json
+
+PackageIdentifier: WiseCleaner.WiseImageX
+PackageVersion: 1.5.1
+InstallerLocale: en-US
+InstallerType: inno
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+FileExtensions:
+- jpeg
+- jpg
+- png
+- webp
+ProductCode: Wise ImageX_is1
+ReleaseDate: 2025-07-07
+AppsAndFeaturesEntries:
+- Publisher: Lespeed Technology Co., Ltd.
+  ProductCode: Wise ImageX_is1
+ElevationRequirement: elevatesSelf
+InstallationMetadata:
+  DefaultInstallLocation: '%ProgramFiles%\Wise\Wise ImageX'
+Dependencies:
+  PackageDependencies:
+  - PackageIdentifier: Microsoft.VCRedist.2015+.x64
+Installers:
+- Architecture: x86

```

```suggestion
- Architecture: x64
```
---
This errors out on x86.
```
Exception during executable launch operation System.AggregateException: One or more errors occurred. (An error occurred trying to start process 'C:\Program Files\Wise\Wise ImageX\imagex.exe' with working directory 'D:\TOOLS'. The specified executable is not a valid application for this OS platform.)
```